django_project/minisass_frontend/src/components/Map/Map.tsx

```diff
@@ -195,145 +205,165 @@ export const Map = forwardRef((props: Interface, ref) => {
     useEffect(() => {
       let mapInstance = map;
 
-      const moveMapToCoordinates = (longitude, latitude) => {
-
-        const geojson = {
-          type: 'FeatureCollection',
-          features: [
-            {
-              type: 'Feature',
-              geometry: {
-                type: 'Point',
-                coordinates: [longitude, latitude]
-              },
-              properties: {
-                name: 'Selected Point'
-              }
+      try {
```

Comment:
Why do we need to add `try` here ?

Reply:
@dimasciput so I have resolved it in the following ,the error is related to the map instance being null or not fully initialized at the time when the useEffect runs. so I made it return if the map is not ready . I have seen the same thing being done in other parts of the code . will this be good ?